community/modules/scheduler/schedmd-slurm-gcp-v6-controller/main.tf

```diff
@@ -40,6 +40,31 @@ locals {
 
 }
 
+locals {
+  synt_suffix       = substr(md5("${var.project_id}${var.deployment_name}"), 0, 5)
+  synth_bucket_name = "${local.slurm_cluster_name}${local.synt_suffix}"
+}
+
+module "bucket" {
+  source  = "terraform-google-modules/cloud-storage/google"
```

Comment:
nit: my personal preference is to use the resource directly (see startup-script as example) instead of cft. I don't really see cft adding value here and it is just one more layer. But we do both in the toolkit so I am fine to leave it.

Reply:
I agree, though this is quick fix without changes to the interfaces, I'm open to change it near future.